turbo.json

```diff
@@ -3,7 +3,7 @@
   "pipeline": {
     "build": {
       "dependsOn": ["^build"],
-      "outputs": ["dist/**", ".next/**", "!.next/cache/**"]
```

Comment:
@cramforce I had to remove this, otherwise the Vercel deployment would go 500 (Cannot find module './node-environment', see https://vercel-storage-next-integration-test-suite-r3he97xxr.vercel.app/ and https://vercel.com/curated-tests/vercel-storage-next-integration-test-suite/D8mYzVf7PywNqHAkqPTRS8pckRPw/logs?page=1&timeline=past30Minutes&startDate=1683796938829&endDate=1683798738829&selectedLogId=1683798727425872742503900000&selectedLogTimestamp=1683798727425&forceEndDate=1683798738829)